data.js

```diff
@@ -0,0 +1,133 @@
+const projectList = [
+  {
+    id: '1',
+    name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release",
+    featuredImage: './assets/Snapshoot-Portfolio.png',
+    desktopImage: './assets/Snapshoot-Portfolio-desktop.png',
+    closeIcon: './assets/Enabled.png',
+    technologies: {
+      ruby: 'Ruby on rails',
+      css: 'css',
+      javascript: 'Javascript',
+      codekit: 'Codekit',
+      github: 'GitHub',
+      bootstrap: 'Bootstrap',
+      terminal: 'Terminal',
+      codepen: 'Codepen',
+    },
+    liveVersion: 'https://fayob.github.io/portfolio-setup-project/',
+    source: 'github.com/fayob/portfolio-setup-project',
+  },
```

Comment:
I created and used one array, I initially created this file to have all the data instead of having them in the javascript file but I was unable to import it to the javascript file, that's why I don't later use it in the project.